create-publication/src/main/java/no/unit/nva/publication/create/CreateResourceHandler.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.publication.create;
```

Comment:
Do not review this class. It is a toy class to verify that things were actually written in Dynamo